build-system/compile/compile.js

```diff
@@ -162,89 +183,7 @@ function compile(entryModuleFilenames, outputDir, outputFilename, options) {
-      'extensions/amp-bind/**/*.js',
-      // Needed to access to Variant interface from other extensions
-      'extensions/amp-experiment/**/*.js',
-      // Needed to access form impl from other extensions
-      'extensions/amp-form/**/*.js',
-      // Needed to access inputmask impl from other extensions
-      'extensions/amp-inputmask/**/*.js',
-      // Needed for AccessService
-      'extensions/amp-access/**/*.js',
-      // Needed for AmpStoryVariableService
-      'extensions/amp-story/**/*.js',
-      // Needed for SubscriptionsService
-      'extensions/amp-subscriptions/**/*.js',
-      // Needed to access UserNotificationManager from other extensions
-      'extensions/amp-user-notification/**/*.js',
-      // Needed for VideoService
-      'extensions/amp-video-service/**/*.js',
-      // Needed to access ConsentPolicyManager from other extensions
-      'extensions/amp-consent/**/*.js',
-      // Needed to access AmpGeo type for service locator
-      'extensions/amp-geo/**/*.js',
-      // Needed for AmpViewerAssistanceService
-      'extensions/amp-viewer-assistance/**/*.js',
-      // Needed for AmpViewerIntegrationVariableService
-      'extensions/amp-viewer-integration/**/*.js',
-      // Needed for amp-smartlinks dep on amp-skimlinks
-      'extensions/amp-skimlinks/0.1/**/*.js',
-      'src/*.js',
-      'src/**/*.js',
-      '!third_party/babel/custom-babel-helpers.js',
-      // Exclude since it's not part of the runtime/extension binaries.
-      '!extensions/amp-access/0.1/amp-login-done.js',
-      'builtins/**.js',
-      'third_party/caja/html-sanitizer.js',
-      'third_party/closure-library/sha384-generated.js',
-      'third_party/css-escape/css-escape.js',
-      'third_party/fuzzysearch/index.js',
-      'third_party/mustache/**/*.js',
-      'third_party/timeagojs/**/*.js',
-      'third_party/vega/**/*.js',
-      'third_party/d3/**/*.js',
-      'third_party/subscriptions-project/*.js',
-      'third_party/webcomponentsjs/ShadowCSS.js',
-      'third_party/react-dates/bundle.js',
-      'third_party/amp-toolbox-cache-url/**/*.js',
-      'third_party/inputmask/**/*.js',
-      'node_modules/dompurify/dist/purify.es.js',
-      'node_modules/promise-pjs/promise.mjs',
-      'node_modules/rrule/dist/esm/src/index.js',
-      'node_modules/set-dom/dist/set-dom.js',
-      'node_modules/web-animations-js/web-animations.install.js',
-      'node_modules/web-activities/activity-ports.js',
-      'node_modules/@ampproject/animations/dist/animations.mjs',
-      'node_modules/@ampproject/worker-dom/dist/amp/main.mjs',
-      'node_modules/document-register-element/build/' +
-        'document-register-element.patched.js',
-      // 'node_modules/core-js/modules/**.js',
-      // Not sure what these files are, but they seem to duplicate code
-      // one level below and confuse the compiler.
-      '!node_modules/core-js/modules/library/**.js',
-    ];
+    const srcs = [...CLOSURE_SRC_GLOBS];
```

Comment:
Curious: why spread operator, and why not `const srcs = CLOSURE_SRC_GLOBS;`?

Reply:
@rsimha i want to clone the array since i dont want the original array mutated. i could also do `CLOSURE_SRC_GLOBS.slice(0)` here